packages/razzle/config/createConfig.js

```diff
@@ -261,17 +239,20 @@ module.exports = (
       }),
     ];
 
+    config.entry = {
+      client: [paths.appClientIndexJs],
+    };
+
     if (IS_DEV) {
       // Setup Webpack Dev Server on port 3001 and
       // specify our client entry point /client/index.js
-      config.entry = {
-        client: [
+      config.entry.client.unshift(
+        ...[
```

Comment:
?!

Reply:
hot entry needs to go in front of the main javascript entry otherwise the magic doesn't work :sake: